lang/scheme48/Portfile

```diff
@@ -20,11 +20,12 @@ master_sites        ${homepage}/${version}/
 extract.suffix      .tgz
 
 checksums           rmd160  81f7f253f4fb9ff7771c44b28f437ebb77756669 \
-                    sha256  9c4921a90e95daee067cd2e9cc0ffe09e118f4da01c0c0198e577c4f47759df4
+                    sha256  9c4921a90e95daee067cd2e9cc0ffe09e118f4da01c0c0198e577c4f47759df4 \
+                    size    3951356
 
-variant universal   {}
-
-#patchfiles patch-Makefile.in.diff
+# event.h should be included from ${worksrcdir}
+# event.h can be installed by libevent
+configure.cppflags-replace -I${prefix}/include -isystem${prefix}/include
```

Comment:
Wouldn't just prefixing the right -I flag fix this?

Reply:
Yes, `configure.cppflag-prepend -I${worksrcpath}` (or something like that) would also solve the problem.
Is there a reason to prefer prepending `-I`?
[Six of one](https://en.wiktionary.org/wiki/six_of_one,_half_a_dozen_of_the_other) as far as I can tell.

Reply:
-i isn't really portable across compilers for one thing, and it's not used much. I'd stick with prepending.

Reply:
Fair point.
I will make the change.

Reply:
With a little research, it seems that it is more common to use `-isystem` than to use `configure.cppflags-prepend`.
I have no strong feelings one way or another, but we might want to stick with the most common fix.

`-isystem`:
pianobar
libdvdnav
libssh
allegro5
cryptlib
alienarena
openjpeg
graphite2
scheme48
mpv
yafc
freetype
pdflib
pinentry

`configure.cppflags-prepend`:
sheepshaver
basiliskii
basiliskii-devel

Reply:
Fine.